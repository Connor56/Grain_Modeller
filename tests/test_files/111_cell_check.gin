opti prop conp full nosymmetry
title
111_cell
end
cell
19.640766 9.793325 6.487915 112.688542 101.0888 104.763598
name 111_cell
fractional
Fe c 0.0 0.0 0.5
Fe c 0.0 0.499999 0.5
Fe c 0.010416 0.249999 0.479166
Fe c 0.046875 0.359378 0.187498
Fe c 0.047538 0.5 0.904921
Fe c 0.059565 0.82131 0.238259
Fe c 0.083333 0.25 0.833332
Fe c 0.083333 0.75 0.833332
Fe c 0.107101 0.67869 0.428405
Fe c 0.119128 1e-06 0.761744
Fe c 0.119792 0.140624 0.479167
Fe c 0.15625 0.250001 0.187499
Fe c 0.166667 1e-06 0.166664
Fe c 0.166667 0.5 0.166664
Fe c 0.177083 0.75 0.145832
Fe c 0.213541 0.859375 0.854162
Fe c 0.214204 0.0 0.571586
Fe c 0.22623 0.321307 0.904921
Fe c 0.25 0.25 0.499999
Fe c 0.25 0.75 0.499998
Fe c 0.27377 0.178694 0.095076
Fe c 0.285795 0.500001 0.428409
Fe c 0.286459 0.640625 0.145834
Fe c 0.322917 0.750001 0.854165
Fe c 0.333333 0.0 0.833332
Fe c 0.333333 0.5 0.833332
Fe c 0.343749 0.249999 0.812499
Fe c 0.380208 0.359376 0.520831
Fe c 0.380872 0.499999 0.238255
Fe c 0.392898 0.821308 0.571589
Fe c 0.416667 0.25 0.166665
Fe c 0.416667 0.75 0.166665
Fe c 0.440435 0.678692 0.761741
Fe c 0.452462 1e-06 0.095076
Fe c 0.453125 0.140624 0.8125
Fe c 0.489584 0.250001 0.520832
Fe c 0.5 0.0 0.499998
Fe c 0.5 0.5 0.499999
Fe c 0.510416 0.749999 0.479166
Fe c 0.546875 0.859376 0.187497
Fe c 0.547538 0.0 0.90492
Fe c 0.559564 0.321305 0.238254
Fe c 0.583333 0.25 0.833332
Fe c 0.583333 0.75 0.833332
Fe c 0.607103 0.178695 0.42841
Fe c 0.619129 0.500002 0.761745
Fe c 0.619792 0.640625 0.479167
Fe c 0.656251 0.750001 0.187498
Fe c 0.666666 0.5 0.166666
Fe c 0.666667 1e-06 0.166665
Fe c 0.677083 0.25 0.145833
Fe c 0.713542 0.359377 0.854166
Fe c 0.714205 0.499999 0.571588
Fe c 0.72623 0.821307 0.904921
Fe c 0.75 0.250001 0.5
Fe c 0.75 0.75 0.499999
Fe c 0.77377 0.678693 0.095077
Fe c 0.785795 1e-06 0.42841
Fe c 0.786459 0.140625 0.145834
Fe c 0.822917 0.250001 0.854166
Fe c 0.833333 0.0 0.833332
Fe c 0.833334 0.500001 0.833333
Fe c 0.84375 0.75 0.812501
Fe c 0.880208 0.859375 0.520832
Fe c 0.880871 0.0 0.238255
Fe c 0.892897 0.321308 0.571589
Fe c 0.916667 0.25 0.166665
Fe c 0.916667 0.75 0.166667
Fe c 0.940436 0.178693 0.761742
Fe c 0.952462 0.500002 0.095079
Fe c 0.953126 0.640626 0.812503
Fe c 0.989584 0.750001 0.520833
Nd c 0.0 1e-06 0.0
Nd c 0.166666 0.5 0.666665
Nd c 0.333333 0.0 0.333332
Nd c 0.5 0.5 0.0
Nd c 0.666666 1e-06 0.666665
Nd c 0.833333 0.5 0.333334
#COORD_END
morse
Nd Nd 0.312 0.945 4.092 12
morse
Nd Fe 0.6036 1.6458 3.188 12
morse
Fe Fe 0.764 1.5995 2.7361 12
dump test_files/111_cell_restart.gin
output cif test_files/111_cell.cif
output xyz test_files/111_cell.xyz
