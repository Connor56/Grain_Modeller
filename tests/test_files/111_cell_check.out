********************************************************************************
*                       GENERAL UTILITY LATTICE PROGRAM                        *
*                                 Julian Gale                                  *
*                       Curtin Institute for Computation                       *
*                    School of Molecular and Life Sciences                     *
*                    Curtin University, Western Australia                      *
********************************************************************************
* Version = 5.2.0 * Last modified =  30th August 2020                          *
********************************************************************************
*  optimise     - perform optimisation run                                     *
*  conp         - constant pressure calculation                                *
*  property     - calculate properties for final geometry                      *
*  nosymmetry   - turn off symmetry after initial structure generation         *
*  full         - generate full unit cell when symmetry is removed             *
********************************************************************************
* 111_cell                                                                     *
********************************************************************************


  Job Started  at 15:51.19  5th November   2020

  Number of CPUs =     1


  Total number of configurations input =      1

********************************************************************************
*  Input for Configuration =   1 : 111_cell                                    *
********************************************************************************

  Formula = Fe72Nd6

  Number of irreducible atoms/shells =      78


  Total number atoms/shells =      78

  Dimensionality = 3               :  Bulk



  Cartesian lattice vectors (Angstroms) :

       19.640766    0.000000    0.000000
       -2.495647    9.470003    0.000000
       -1.247822   -2.916810    5.659346

  Cell parameters (Angstroms/Degrees):

  a =      19.6408    alpha = 112.6885
  b =       9.7933    beta  = 101.0888
  c =       6.4879    gamma = 104.7636

  Initial cell volume =        1052.627781 Angs**3

  Temperature of configuration =   0.00     K

  Pressure of configuration =         0.000 GPa

  Fractional coordinates of asymmetric unit :

--------------------------------------------------------------------------------
   No.  Atomic       x           y          z         Charge      Occupancy
        Label      (Frac)      (Frac)     (Frac)        (e)         (Frac)
--------------------------------------------------------------------------------
      1 Fe    c    0.000000    0.000000    0.500000     0.00000    1.000000
      2 Fe    c    0.010416 *  0.249999 *  0.479166 *   0.00000    1.000000
      3 Fe    c    0.046875 *  0.359378 *  0.187498 *   0.00000    1.000000
      4 Fe    c    0.047538 *  0.500000 *  0.904921 *   0.00000    1.000000
      5 Fe    c    0.059565 *  0.821310 *  0.238259 *   0.00000    1.000000
      6 Fe    c    0.083333 *  0.250000 *  0.833332 *   0.00000    1.000000
      7 Fe    c    0.083333 *  0.750000 *  0.833332 *   0.00000    1.000000
      8 Fe    c    0.107101 *  0.678690 *  0.428405 *   0.00000    1.000000
      9 Fe    c    0.119128 *  0.000001 *  0.761744 *   0.00000    1.000000
     10 Fe    c    0.119792 *  0.140624 *  0.479167 *   0.00000    1.000000
     11 Fe    c    0.156250 *  0.250001 *  0.187499 *   0.00000    1.000000
     12 Fe    c    0.166667 *  0.000001 *  0.166664 *   0.00000    1.000000
     13 Fe    c    0.166667 *  0.500000 *  0.166664 *   0.00000    1.000000
     14 Fe    c    0.177083 *  0.750000 *  0.145832 *   0.00000    1.000000
     15 Fe    c    0.213541 *  0.859375 *  0.854162 *   0.00000    1.000000
     16 Fe    c    0.214204 *  0.999998 *  0.571586 *   0.00000    1.000000
     17 Fe    c    0.226230 *  0.321307 *  0.904921 *   0.00000    1.000000
     18 Fe    c    0.250000 *  0.250000 *  0.499999 *   0.00000    1.000000
     19 Fe    c    0.250000 *  0.750000 *  0.499998 *   0.00000    1.000000
     20 Fe    c    0.273770 *  0.178694 *  0.095076 *   0.00000    1.000000
     21 Fe    c    0.285795 *  0.500001 *  0.428409 *   0.00000    1.000000
     22 Fe    c    0.286459 *  0.640625 *  0.145834 *   0.00000    1.000000
     23 Fe    c    0.322917 *  0.750001 *  0.854165 *   0.00000    1.000000
     24 Fe    c    0.333333 *  0.000000 *  0.833332 *   0.00000    1.000000
     25 Fe    c    0.333333 *  0.500000 *  0.833332 *   0.00000    1.000000
     26 Fe    c    0.343749 *  0.249999 *  0.812499 *   0.00000    1.000000
     27 Fe    c    0.380208 *  0.359376 *  0.520831 *   0.00000    1.000000
     28 Fe    c    0.380872 *  0.499999 *  0.238255 *   0.00000    1.000000
     29 Fe    c    0.392898 *  0.821308 *  0.571589 *   0.00000    1.000000
     30 Fe    c    0.416667 *  0.250000 *  0.166665 *   0.00000    1.000000
     31 Fe    c    0.416667 *  0.750000 *  0.166665 *   0.00000    1.000000
     32 Fe    c    0.440435 *  0.678692 *  0.761741 *   0.00000    1.000000
     33 Fe    c    0.452462 *  0.000001 *  0.095076 *   0.00000    1.000000
     34 Fe    c    0.453125 *  0.140624 *  0.812500 *   0.00000    1.000000
     35 Fe    c    0.489584 *  0.250001 *  0.520832 *   0.00000    1.000000
     36 Fe    c    0.500000 *  0.000000 *  0.499998 *   0.00000    1.000000
     37 Fe    c    0.500000 *  0.500000 *  0.499999 *   0.00000    1.000000
     38 Fe    c    0.510416 *  0.749999 *  0.479166 *   0.00000    1.000000
     39 Fe    c    0.546875 *  0.859376 *  0.187497 *   0.00000    1.000000
     40 Fe    c    0.547538 *  0.999999 *  0.904920 *   0.00000    1.000000
     41 Fe    c    0.559564 *  0.321305 *  0.238254 *   0.00000    1.000000
     42 Fe    c    0.583333 *  0.250000 *  0.833332 *   0.00000    1.000000
     43 Fe    c    0.583333 *  0.750000 *  0.833332 *   0.00000    1.000000
     44 Fe    c    0.607103 *  0.178695 *  0.428410 *   0.00000    1.000000
     45 Fe    c    0.619129 *  0.500002 *  0.761745 *   0.00000    1.000000
     46 Fe    c    0.619792 *  0.640625 *  0.479167 *   0.00000    1.000000
     47 Fe    c    0.656251 *  0.750001 *  0.187498 *   0.00000    1.000000
     48 Fe    c    0.666666 *  0.500000 *  0.166666 *   0.00000    1.000000
     49 Fe    c    0.666667 *  0.000001 *  0.166665 *   0.00000    1.000000
     50 Fe    c    0.677083 *  0.250000 *  0.145833 *   0.00000    1.000000
     51 Fe    c    0.713542 *  0.359377 *  0.854166 *   0.00000    1.000000
     52 Fe    c    0.714205 *  0.499999 *  0.571588 *   0.00000    1.000000
     53 Fe    c    0.726230 *  0.821307 *  0.904921 *   0.00000    1.000000
     54 Fe    c    0.750000 *  0.250001 *  0.500000 *   0.00000    1.000000
     55 Fe    c    0.750000 *  0.750000 *  0.499999 *   0.00000    1.000000
     56 Fe    c    0.773770 *  0.678693 *  0.095077 *   0.00000    1.000000
     57 Fe    c    0.785795 *  0.000001 *  0.428410 *   0.00000    1.000000
     58 Fe    c    0.786459 *  0.140625 *  0.145834 *   0.00000    1.000000
     59 Fe    c    0.822917 *  0.250001 *  0.854166 *   0.00000    1.000000
     60 Fe    c    0.833333 *  0.000000 *  0.833332 *   0.00000    1.000000
     61 Fe    c    0.833334 *  0.500001 *  0.833333 *   0.00000    1.000000
     62 Fe    c    0.843750 *  0.750000 *  0.812501 *   0.00000    1.000000
     63 Fe    c    0.880208 *  0.859375 *  0.520832 *   0.00000    1.000000
     64 Fe    c    0.880871 *  0.999999 *  0.238255 *   0.00000    1.000000
     65 Fe    c    0.892897 *  0.321308 *  0.571589 *   0.00000    1.000000
     66 Fe    c    0.916667 *  0.250000 *  0.166665 *   0.00000    1.000000
     67 Fe    c    0.916667 *  0.750000 *  0.166667 *   0.00000    1.000000
     68 Fe    c    0.940436 *  0.178693 *  0.761742 *   0.00000    1.000000
     69 Fe    c    0.952462 *  0.500002 *  0.095079 *   0.00000    1.000000
     70 Fe    c    0.953126 *  0.640626 *  0.812503 *   0.00000    1.000000
     71 Fe    c    0.989584 *  0.750001 *  0.520833 *   0.00000    1.000000
     72 Fe    c    0.999999 *  0.499999 *  0.500000 *   0.00000    1.000000
     73 Nd    c    0.000000 *  0.000001 *  0.000000 *   0.00000    1.000000
     74 Nd    c    0.166666 *  0.500000 *  0.666665 *   0.00000    1.000000
     75 Nd    c    0.333333 *  0.000000 *  0.333332 *   0.00000    1.000000
     76 Nd    c    0.500000 *  0.500000 *  0.000000 *   0.00000    1.000000
     77 Nd    c    0.666666 *  0.000001 *  0.666665 *   0.00000    1.000000
     78 Nd    c    0.833333 *  0.500000 *  0.333334 *   0.00000    1.000000
--------------------------------------------------------------------------------



********************************************************************************
*  General input information                                                   *
********************************************************************************

  Species output for all configurations :

--------------------------------------------------------------------------------
  Species    Type    Atomic    Atomic    Charge       Radii (Angs)     Library
                     Number     Mass       (e)     Cova   Ionic  VDW   Symbol
--------------------------------------------------------------------------------
    Fe       Core       26      55.85   0.000000   1.340  0.000  2.000
    Nd       Core       60     144.24   0.000000   1.810  0.000  2.700
--------------------------------------------------------------------------------


  Lattice summation method               =    Ewald          (3-D)
                                         =    Parry          (2-D)
                                         =    Saunders et al (1-D)
  Accuracy factor for lattice sums       =   12.000


  Analytic derivatives to be used

  Time limit = Infinity

  Maximum range for interatomic potentials =    100000.000000 Angstroms

  General interatomic potentials :

--------------------------------------------------------------------------------
Atom  Types   Potential      Parameter       Value         Units   Cutoffs(Ang)
  1     2                                                            Min /  Max
--------------------------------------------------------------------------------
Nd   c Nd   c Morse         Morse  De       0.31200000    eV        0.000 12.000
                            Morse  a0       0.94500000    Ang^-1
                            Morse  r0        4.0920000    Ang
--------------------------------------------------------------------------------
Fe   c Nd   c Morse         Morse  De       0.60360000    eV        0.000 12.000
                            Morse  a0        1.6458000    Ang^-1
                            Morse  r0        3.1880000    Ang
--------------------------------------------------------------------------------
Fe   c Fe   c Morse         Morse  De       0.76400000    eV        0.000 12.000
                            Morse  a0        1.5995000    Ang^-1
                            Morse  r0        2.7361000    Ang
--------------------------------------------------------------------------------

********************************************************************************
*  Output for configuration   1 : 111_cell                                     *
********************************************************************************


  Components of energy :

--------------------------------------------------------------------------------
  Interatomic potentials     =        -470.18283435 eV
  Monopole - monopole (real) =           0.00000000 eV
  Monopole - monopole (recip)=           0.00000000 eV
  Monopole - monopole (total)=           0.00000000 eV
--------------------------------------------------------------------------------
  Total lattice energy       =        -470.18283435 eV
--------------------------------------------------------------------------------
  Total lattice energy       =          -45365.4561 kJ/(mole unit cells)
--------------------------------------------------------------------------------


  Number of variables =      237

  Maximum number of calculations  =          1000
  Maximum Hessian update interval =            10
  Maximum step size               =   1.000000000
  Maximum parameter tolerance     =   0.000010000
  Maximum function  tolerance     =   0.000010000
  Maximum gradient  tolerance     =   0.001000000
  Maximum gradient  component     =   0.010000000

  Cell parameters to be optimised using strains

  Newton-Raphson optimiser to be used

  BFGS hessian update to be used

  Lower half triangular hessian to be used

  Start of bulk optimisation :

  Cycle:      0 Energy:      -470.182834  Gnorm:      0.000125  CPU:    0.090

  XYZ File written as test_files/111_cell.xyz



  **** Optimisation achieved ****


  Final energy =    -470.18283435 eV
  Final Gnorm  =       0.00012512

  Components of energy :

--------------------------------------------------------------------------------
  Interatomic potentials     =        -470.18283435 eV
  Monopole - monopole (real) =           0.00000000 eV
  Monopole - monopole (recip)=           0.00000000 eV
  Monopole - monopole (total)=           0.00000000 eV
--------------------------------------------------------------------------------
  Total lattice energy       =        -470.18283435 eV
--------------------------------------------------------------------------------
  Total lattice energy       =          -45365.4561 kJ/(mole unit cells)
--------------------------------------------------------------------------------

  Final fractional coordinates of atoms :

--------------------------------------------------------------------------------
   No.  Atomic        x           y          z          Radius
        Label       (Frac)      (Frac)     (Frac)       (Angs)
--------------------------------------------------------------------------------
     1  Fe    c     0.000000    0.000000    0.500000    0.000000
     2  Fe    c     0.010416    0.249999    0.479166    0.000000
     3  Fe    c     0.046875    0.359378    0.187498    0.000000
     4  Fe    c     0.047538    0.500000    0.904921    0.000000
     5  Fe    c     0.059565    0.821310    0.238259    0.000000
     6  Fe    c     0.083333    0.250000    0.833332    0.000000
     7  Fe    c     0.083333    0.750000    0.833332    0.000000
     8  Fe    c     0.107101    0.678690    0.428405    0.000000
     9  Fe    c     0.119128    0.000001    0.761744    0.000000
    10  Fe    c     0.119792    0.140624    0.479167    0.000000
    11  Fe    c     0.156250    0.250001    0.187499    0.000000
    12  Fe    c     0.166667    0.000001    0.166664    0.000000
    13  Fe    c     0.166667    0.500000    0.166664    0.000000
    14  Fe    c     0.177083    0.750000    0.145832    0.000000
    15  Fe    c     0.213541    0.859375    0.854162    0.000000
    16  Fe    c     0.214204    0.999998    0.571586    0.000000
    17  Fe    c     0.226230    0.321307    0.904921    0.000000
    18  Fe    c     0.250000    0.250000    0.499999    0.000000
    19  Fe    c     0.250000    0.750000    0.499998    0.000000
    20  Fe    c     0.273770    0.178694    0.095076    0.000000
    21  Fe    c     0.285795    0.500001    0.428409    0.000000
    22  Fe    c     0.286459    0.640625    0.145834    0.000000
    23  Fe    c     0.322917    0.750001    0.854165    0.000000
    24  Fe    c     0.333333    0.000000    0.833332    0.000000
    25  Fe    c     0.333333    0.500000    0.833332    0.000000
    26  Fe    c     0.343749    0.249999    0.812499    0.000000
    27  Fe    c     0.380208    0.359376    0.520831    0.000000
    28  Fe    c     0.380872    0.499999    0.238255    0.000000
    29  Fe    c     0.392898    0.821308    0.571589    0.000000
    30  Fe    c     0.416667    0.250000    0.166665    0.000000
    31  Fe    c     0.416667    0.750000    0.166665    0.000000
    32  Fe    c     0.440435    0.678692    0.761741    0.000000
    33  Fe    c     0.452462    0.000001    0.095076    0.000000
    34  Fe    c     0.453125    0.140624    0.812500    0.000000
    35  Fe    c     0.489584    0.250001    0.520832    0.000000
    36  Fe    c     0.500000    0.000000    0.499998    0.000000
    37  Fe    c     0.500000    0.500000    0.499999    0.000000
    38  Fe    c     0.510416    0.749999    0.479166    0.000000
    39  Fe    c     0.546875    0.859376    0.187497    0.000000
    40  Fe    c     0.547538    0.999999    0.904920    0.000000
    41  Fe    c     0.559564    0.321305    0.238254    0.000000
    42  Fe    c     0.583333    0.250000    0.833332    0.000000
    43  Fe    c     0.583333    0.750000    0.833332    0.000000
    44  Fe    c     0.607103    0.178695    0.428410    0.000000
    45  Fe    c     0.619129    0.500002    0.761745    0.000000
    46  Fe    c     0.619792    0.640625    0.479167    0.000000
    47  Fe    c     0.656251    0.750001    0.187498    0.000000
    48  Fe    c     0.666666    0.500000    0.166666    0.000000
    49  Fe    c     0.666667    0.000001    0.166665    0.000000
    50  Fe    c     0.677083    0.250000    0.145833    0.000000
    51  Fe    c     0.713542    0.359377    0.854166    0.000000
    52  Fe    c     0.714205    0.499999    0.571588    0.000000
    53  Fe    c     0.726230    0.821307    0.904921    0.000000
    54  Fe    c     0.750000    0.250001    0.500000    0.000000
    55  Fe    c     0.750000    0.750000    0.499999    0.000000
    56  Fe    c     0.773770    0.678693    0.095077    0.000000
    57  Fe    c     0.785795    0.000001    0.428410    0.000000
    58  Fe    c     0.786459    0.140625    0.145834    0.000000
    59  Fe    c     0.822917    0.250001    0.854166    0.000000
    60  Fe    c     0.833333    0.000000    0.833332    0.000000
    61  Fe    c     0.833334    0.500001    0.833333    0.000000
    62  Fe    c     0.843750    0.750000    0.812501    0.000000
    63  Fe    c     0.880208    0.859375    0.520832    0.000000
    64  Fe    c     0.880871    0.999999    0.238255    0.000000
    65  Fe    c     0.892897    0.321308    0.571589    0.000000
    66  Fe    c     0.916667    0.250000    0.166665    0.000000
    67  Fe    c     0.916667    0.750000    0.166667    0.000000
    68  Fe    c     0.940436    0.178693    0.761742    0.000000
    69  Fe    c     0.952462    0.500002    0.095079    0.000000
    70  Fe    c     0.953126    0.640626    0.812503    0.000000
    71  Fe    c     0.989584    0.750001    0.520833    0.000000
    72  Fe    c     0.999999    0.499999    0.500000    0.000000
    73  Nd    c     0.000000    0.000001    0.000000    0.000000
    74  Nd    c     0.166666    0.500000    0.666665    0.000000
    75  Nd    c     0.333333    0.000000    0.333332    0.000000
    76  Nd    c     0.500000    0.500000    0.000000    0.000000
    77  Nd    c     0.666666    0.000001    0.666665    0.000000
    78  Nd    c     0.833333    0.500000    0.333334    0.000000
--------------------------------------------------------------------------------

  Final Cartesian lattice vectors (Angstroms) :

       19.640766    3.400000    2.400000
       -2.495647    9.470003    1.470000
       -1.247822   -2.916810    5.659346


  Final cell parameters and derivatives :

--------------------------------------------------------------------------------
       a           19.640766 Angstrom     dE/de1(xx)    -0.003739 eV/strain
       b            9.793325 Angstrom     dE/de2(yy)    -0.004306 eV/strain
       c            6.487915 Angstrom     dE/de3(zz)    -0.003624 eV/strain
       alpha      112.688542 Degrees      dE/de4(yz)    -0.000799 eV/strain
       beta       101.088800 Degrees      dE/de5(xz)    -0.001222 eV/strain
       gamma      104.763598 Degrees      dE/de6(xy)    -0.000638 eV/strain
--------------------------------------------------------------------------------

  Primitive cell volume =          1052.627781 Angs**3

  Density of cell =      7.708884 g/cm**3

  Non-primitive cell volume =          1052.627781 Angs**3


  Final internal derivatives :

--------------------------------------------------------------------------------
   No.  Atomic          a             b             c           Radius
        Label          (eV)          (eV)          (eV)        (eV/Angs)
--------------------------------------------------------------------------------
      1 Fe    c       0.000000      0.000000      0.000000      0.000000
      2 Fe    c       0.000975     -0.001863      0.001062      0.000000
      3 Fe    c       0.003075      0.001954     -0.001255      0.000000
      4 Fe    c      -0.000622      0.001077     -0.000841      0.000000
      5 Fe    c       0.000725     -0.001265      0.001724      0.000000
      6 Fe    c      -0.000196     -0.000460     -0.000287      0.000000
      7 Fe    c      -0.004938     -0.000045     -0.000050      0.000000
      8 Fe    c      -0.005995      0.000912     -0.001090      0.000000
      9 Fe    c      -0.003064      0.000241      0.000467      0.000000
     10 Fe    c       0.000447     -0.000857      0.000543      0.000000
     11 Fe    c      -0.003502      0.001217     -0.000421      0.000000
     12 Fe    c       0.000205      0.001232     -0.000532      0.000000
     13 Fe    c       0.002673     -0.001257      0.000008      0.000000
     14 Fe    c       0.002699     -0.000642      0.000039      0.000000
     15 Fe    c       0.002990     -0.000652     -0.000511      0.000000
     16 Fe    c      -0.000743     -0.000618     -0.000158      0.000000
     17 Fe    c      -0.003200      0.000013      0.000252      0.000000
     18 Fe    c       0.001529     -0.000841      0.000802      0.000000
     19 Fe    c       0.002259      0.000074     -0.000225      0.000000
     20 Fe    c       0.005043      0.000583     -0.000936      0.000000
     21 Fe    c      -0.000285      0.001281     -0.000729      0.000000
     22 Fe    c      -0.001220      0.000350      0.000753      0.000000
     23 Fe    c      -0.001339      0.002064     -0.000706      0.000000
     24 Fe    c      -0.001457      0.000151      0.000208      0.000000
     25 Fe    c       0.000408      0.000369     -0.000390      0.000000
     26 Fe    c      -0.000618     -0.001747      0.000599      0.000000
     27 Fe    c      -0.001109      0.000870     -0.000527      0.000000
     28 Fe    c       0.004304     -0.001811      0.000587      0.000000
     29 Fe    c       0.004583     -0.001978      0.000544      0.000000
     30 Fe    c       0.001596      0.000148     -0.000878      0.000000
     31 Fe    c       0.001666      0.000050      0.000213      0.000000
     32 Fe    c      -0.005236      0.002271     -0.000852      0.000000
     33 Fe    c       0.002775      0.000794     -0.000883      0.000000
     34 Fe    c      -0.001849     -0.000434      0.001688      0.000000
     35 Fe    c       0.001287      0.001208     -0.000589      0.000000
     36 Fe    c      -0.000388     -0.000753      0.000247      0.000000
     37 Fe    c      -0.003376      0.001189     -0.000181      0.000000
     38 Fe    c       0.000905     -0.002605      0.001048      0.000000
     39 Fe    c       0.001681     -0.000376     -0.000765      0.000000
     40 Fe    c      -0.000987     -0.000195     -0.000198      0.000000
     41 Fe    c       0.005256     -0.005716      0.001388      0.000000
     42 Fe    c      -0.004919     -0.000116      0.000278      0.000000
     43 Fe    c       0.000354     -0.000007     -0.000444      0.000000
     44 Fe    c      -0.002294      0.004993     -0.002145      0.000000
     45 Fe    c       0.001668      0.000427      0.000387      0.000000
     46 Fe    c       0.001282      0.000775     -0.000166      0.000000
     47 Fe    c       0.001670      0.000058     -0.000648      0.000000
     48 Fe    c      -0.003652      0.000563      0.000603      0.000000
     49 Fe    c       0.003027      0.001017     -0.000574      0.000000
     50 Fe    c       0.000999     -0.001527      0.001156      0.000000
     51 Fe    c       0.005688     -0.000412     -0.001165      0.000000
     52 Fe    c       0.001914     -0.001723     -0.000036      0.000000
     53 Fe    c      -0.005560      0.000544      0.000172      0.000000
     54 Fe    c      -0.001695      0.001278      0.000425      0.000000
     55 Fe    c      -0.000284      0.000848     -0.000499      0.000000
     56 Fe    c       0.004868     -0.001101      0.000254      0.000000
     57 Fe    c       0.002077      0.000227     -0.000969      0.000000
     58 Fe    c       0.002203      0.000052     -0.000029      0.000000
     59 Fe    c      -0.002364      0.000926     -0.000195      0.000000
     60 Fe    c      -0.000555      0.000088     -0.000034      0.000000
     61 Fe    c       0.001943      0.001584     -0.000445      0.000000
     62 Fe    c       0.004941     -0.002804      0.000850      0.000000
     63 Fe    c      -0.000967     -0.001375      0.000191      0.000000
     64 Fe    c      -0.004576      0.000192      0.000579      0.000000
     65 Fe    c      -0.002698      0.001138     -0.000143      0.000000
     66 Fe    c       0.000910      0.000765      0.000112      0.000000
     67 Fe    c       0.000165     -0.001304     -0.000134      0.000000
     68 Fe    c       0.003029      0.000512     -0.000643      0.000000
     69 Fe    c       0.000324      0.000884      0.000217      0.000000
     70 Fe    c       0.002985     -0.000141      0.000205      0.000000
     71 Fe    c       0.001062      0.001779     -0.000596      0.000000
     72 Fe    c      -0.002355     -0.001490      0.000980      0.000000
     73 Nd    c       0.000981      0.000174     -0.000264      0.000000
     74 Nd    c      -0.004756      0.000287      0.000157      0.000000
     75 Nd    c      -0.003263     -0.000479      0.000505      0.000000
     76 Nd    c      -0.001025     -0.000659      0.001378      0.000000
     77 Nd    c      -0.005881      0.001775     -0.000101      0.000000
     78 Nd    c      -0.004832     -0.000877      0.001124      0.000000
--------------------------------------------------------------------------------
  Maximum abs         0.005995      0.005716      0.002145      0.000000
--------------------------------------------------------------------------------



  Elastic Constant Matrix: (Units=GPa)

-------------------------------------------------------------------------------
  Indices      1         2         3         4         5         6
-------------------------------------------------------------------------------
       1    515.1005  185.8094  213.1113  -17.1937  -52.6601   49.4904
       2    185.8094  523.0250  203.9337   -9.9400   41.0540  -45.3390
       3    213.1113  203.9337  494.4506   25.0606   10.7206   -4.8058
       4    -17.1937   -9.9400   25.0606  167.7061   -8.0991   36.6000
       5    -52.6601   41.0540   10.7206   -8.0991  183.1730  -27.6027
       6     49.4904  -45.3390   -4.8058   36.6000  -27.6027  162.2455
-------------------------------------------------------------------------------


  Elastic Compliance Matrix: (Units=1/GPa)

-------------------------------------------------------------------------------
  Indices      1         2         3         4         5         6
-------------------------------------------------------------------------------
       1    0.002850 -0.000791 -0.000967  0.000678  0.000915 -0.001116
       2   -0.000791  0.002590 -0.000703 -0.000038 -0.000641  0.000844
       3   -0.000967 -0.000703  0.002770 -0.000628 -0.000269  0.000277
       4    0.000678 -0.000038 -0.000628  0.006498  0.000278 -0.001654
       5    0.000915 -0.000641 -0.000269  0.000278  0.005967  0.000486
       6   -0.001116  0.000844  0.000277 -0.001654  0.000486  0.007204
-------------------------------------------------------------------------------

  Mechanical properties :

-------------------------------------------------------------------------------
  Convention :                    Reuss         Voigt         Hill
-------------------------------------------------------------------------------
  Bulk  Modulus (GPa)     =     304.24970     304.25389     304.25180
  Shear Modulus (GPa)     =     147.49792     164.60636     156.05214
-------------------------------------------------------------------------------
  Velocity S-wave (km/s)  =       4.37419       4.62091       4.49924
  Velocity P-wave (km/s)  =       8.06094       8.24247       8.15221
-------------------------------------------------------------------------------
  Compressibility (1/GPa) =    0.00328677
-------------------------------------------------------------------------------
  Stress axis :                     x             y             z
-------------------------------------------------------------------------------
  Youngs Moduli (GPa)     =     350.82804     386.10267     361.04857
-------------------------------------------------------------------------------
  Poissons Ratio (x)      =                     0.30549       0.34922
  Poissons Ratio (y)      =       0.27758                     0.25389
  Poissons Ratio (z)      =       0.33934       0.27151
-------------------------------------------------------------------------------


  Piezoelectric Strain Matrix: (Units=C/m**2)

-------------------------------------------------------------------------------
  Indices      1         2         3         4         5         6
-------------------------------------------------------------------------------
       x    -0.00000  -0.00000  -0.00000  -0.00000  -0.00000  -0.00000
       y    -0.00000  -0.00000  -0.00000  -0.00000  -0.00000  -0.00000
       z    -0.00000  -0.00000  -0.00000  -0.00000  -0.00000  -0.00000
-------------------------------------------------------------------------------


  Piezoelectric Stress Matrix: (Units=10**-11 C/N)

-------------------------------------------------------------------------------
  Indices      1         2         3         4         5         6
-------------------------------------------------------------------------------
       x     0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
       y     0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
       z     0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
-------------------------------------------------------------------------------


  Static dielectric constant tensor :

-------------------------------------------------------------------------------
              x         y         z
-------------------------------------------------------------------------------
       x     1.00000   0.00000   0.00000
       y     0.00000   1.00000   0.00000
       z     0.00000   0.00000   1.00000
--------------------------------------------------------------------------------

  Static refractive indices :

-------------------------------------------------------------------------------
    1 =    1.00000      2 =    1.00000      3 =    1.00000
-------------------------------------------------------------------------------


  Time to end of optimisation =       0.1846 seconds


  Peak dynamic memory used =       3.19 MB


  Timing analysis for GULP :

--------------------------------------------------------------------------------
  Task / Subroutine                                          Time (Seconds)
--------------------------------------------------------------------------------
  Calculation of real space energy and derivatives                0.1716
--------------------------------------------------------------------------------
  Total CPU time                                                  0.1847
--------------------------------------------------------------------------------


  Dump file written as test_files/111_cell_restart.gin
  CIF file written as test_files/111_cell.cif


  Job Finished at 15:51.19  5th November   2020
